lib/pbench/test/unit/server/query_apis/test_datasets_list.py

```diff
@@ -55,20 +75,25 @@ def test_missing_keys(self, client, server_config, keys):
         however, Pbench will silently ignore any additional keys that are
         specified.
        """
-        response = client.post(f"{server_config.rest_uri}/datasets/list", json=keys)
+        response = client.post(
+            f"{server_config.rest_uri}/datasets/list",
+            headers={"Authorization": "Bearer " + pbench_token},
+            json=keys,
+        )
         assert response.status_code == HTTPStatus.BAD_REQUEST
-        missing = [k for k in ("user", "controller", "start", "end") if k not in keys]
+        missing = [k for k in ("controller", "start", "end") if k not in keys]
```

Comment:
See both comments in `TestDatasetsDetail:test_missing_keys()` -- they apply here, too.